src/transformers/models/groupvit/modeling_tf_groupvit.py

```diff
@@ -407,7 +407,7 @@ def call(self, image_tokens: tf.Tensor, group_tokens: tf.Tensor, training: Optio
         return new_image_tokens, attention
 
 
-# Copied from transformers.models.vit.modeling_tf_vit.TFViTPatchEmbeddings with ViT->GroupViT
+# Adapted from transformers.models.vit.modeling_tf_vit.TFViTPatchEmbeddings with ViT->GroupViT
```

Comment:
Why are you using Adapted here? You don't want to leverage Copied from?

Reply:
There is a minor code addition:

```
        # hidden_size is a member as it will be required in the call method
        self.hidden_size = config.hidden_size
```

This leads me to add the `# Adapted from` instead of `# Copied from`